package.json

```diff
@@ -54,6 +54,7 @@
     "gulp-eslint": "5.0.0",
     "gulp-sass-lint": "1.4.0",
     "html-webpack-plugin": "3.0.6",
+    "husky": "1.3.1",
```

Comment:
This version of husky requires putting your git hooks (precommit) in a property called “husky”. Putting “precommit” in the “scripts” property will no longer work.

Reply:
I installed this version and I didn't need to create the husky property for it to work. And also my precommit that I defined in the script seems to be working fine. It's being triggered when I do a git commit.

Reply:
Ah ok cool. My bad. I recently upgraded and just followed the instructions on the NPM package page. I guess they made it backwards compatible.

I haven’t tested it as I’m not on my computer at the moment. But if it works for you then it should be good.